### PR TITLE
Apple.iPod13Update version 1.3

### PR DESCRIPTION
**Pull request by upintheairsheep**
Here we go, reopening all stale pulls and fixing them.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106244)

### CODE REVIEW

File: manifests/a/Apple/iPod13Update/1.3/Apple.iPod13Update.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.iPod13Update
+PackageVersion: 1.3
+PackageLocale: en-US
+Publisher: Apple Computer, Inc.
+PackageName: iPod Software 1.3 Updater
+License: Propretary
+Copyright: 751
```
Comment (Trenly) [inline manifests/a/Apple/iPod13Update/1.3/Apple.iPod13Update.locale.en-US.yaml:10]: This doesn't look like a valid copyright

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 2023-05-08 
> 19:24:10.711 [CLI ] ShellExecute installer failed: 4294967293 
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230509022410

File: manifests/a/Apple/iPod13Update/1.3/Apple.iPod13Update.locale.en-US.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/a/Apple/iPod13Update/1.3/Apple.iPod13Update.locale.en-US.yaml:10]:
```suggestion
Publisher: Apple Computer, Inc.
PublisherUrl: https://www.apple.com/
PublisherSupportUrl: https://support.apple.com/
PrivacyUrl:   https://www.apple.com/legal/privacy/en-ww/
PackageName: iPod Software 1.3 Updater
PackageUrl: https://support.apple.com/kb/DL729?viewlocale=en_US&locale=en_US
License: Propretary
Copyright: ©2003 Apple Computer, Inc.
```

Comment (stephengillie) [reply manifests/a/Apple/iPod13Update/1.3/Apple.iPod13Update.locale.en-US.yaml:10]: Copyright was found in the `readme.doc` file accompanying the package.

**Comment by stephengillie** [comment]
This package is failing to install correctly, and I'm having difficulty determining why.

**Comment by stephengillie** [comment]
>  Manifest Error: Duplicate field found in the manifest. [Publisher]

File: manifests/a/Apple/iPod13Update/1.3/Apple.iPod13Update.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.iPod13Update
+PackageVersion: 1.3
+PackageLocale: en-US
+Publisher: Apple Computer, Inc.
+Publisher: Apple Computer, Inc.
```
Comment (stephengillie) [inline manifests/a/Apple/iPod13Update/1.3/Apple.iPod13Update.locale.en-US.yaml:8]:
```suggestion
```

**Comment by upintheairsheep** [comment]
Merge?

**Comment by stephengillie** [comment]
Running interactive, it asks to reboot. It's the only guess I have. 
image

**Comment by upintheairsheep** [comment]
> Running interactive, it asks to reboot. It's the only guess I have. image

Interactive only?

**Comment by upintheairsheep** [comment]
???

**Comment by stephengillie** [comment]
This is unexpected - WinGet fails due to an installer error, but the installer logged success to the Application Log: 
> Windows Installer installed the product. Product Name: iPod Software 1.3 Updater. Product Version: 1.3. Product Language: 1033. Manufacturer: Apple Computer, Inc.. Installation success or error status: 0.

**Comment by stephengillie** [comment]
> This is unexpected - WinGet fails due to an installer error, but the installer logged success to the Application Log:
> 
> > Windows Installer installed the product. Product Name: iPod Software 1.3 Updater. Product Version: 1.3. Product Language: 1033. Manufacturer: Apple Computer, Inc.. Installation success or error status: 0.

Re-running this, the installer appears to return 0 regardless of install outcome. The installer's silent installer switches don't result in a successful install.

**Pull request closed without merging**